test/evaluate/errors.test.ts

```diff
@@ -94,4 +94,49 @@ suite('Evaluation errors', () => {
       );
     });
   });
+
+  suite('Multiple errors', () => {
+    test('Evaluation errors are collected', async () => {
+      // GIVEN
+      const template = {
+        Resources: {
+          SiteDistribution: {
+            Type: 'aws-cdk-lib.aws_cloudfront.Distribution',
+            Properties: {
+              certificate: {
+                Ref: 'SiteBucket', // this should be SiteCertificate
+              },
+              defaultRootObject: 'index.html',
+              domainNames: [
+                {
+                  Ref: 'DomainName',
+                },
+              ],
+              minimumProtocolVersion: 'TLS_V1_2_2021',
+              defaultBehavior: {
+                origin: {
+                  'aws-cdk-lib.aws_cloudfront_origins.S3Origin': [
+                    { Ref: 'SiteCertificate' }, // this should be SiteBucket
+                    {
+                      originAccessIdentity: {
+                        Ref: 'CloudFrontOAI',
+                      },
+                    },
+                  ],
+                },
+              },
+            },
+          },
+        },
+      };
+
+      // THEN
+      const synth = Testing.synth(await Template.fromObject(template), {
+        validateTemplate: false,
+      });
+      await expect(synth).rejects.toThrow(
+        'Expected aws-cdk-lib.aws_s3.IBucket, got: "MyBucket'
```

Comment:
I would expect two messages here. But I'm obviously missing something 🤔

Reply:
Nah you're right. I forgot to push something. 🤦‍♂️